### ISSUE TITLE
line breaks not being registered by screen readers

### ISSUE THREAD

**Issue by crentava** [QuantumBadger/RedReader#813]
hello
this app is one of the most accessible Reddit clients I have ever used. Everything is working well regarding accessibility with only one drawback
if you read a post from a subreddit like r/nosleep with a screen reader you will only hear it be read only on one line. Just to make sure it wasn’t the poster not adding line breaks I used the web client to read the same post. When I read the post on the web client it was split into multiple lines like it should’ve been. Is there anyway that this can be fixed? Thank you for bringing this app to us blind users

**Comment by QuantumBadger** [comment]
Thanks for the report, I'll look into this.

When you say that it's read as one line, do you mean that there isn't a long enough time delay between each line? Also, are you using TalkBack as the screen reader?

**Comment by crentava** [comment]
what I mean by saying it reads as if it was only one line is for example if someone were to type “hello
how’s it going?” it would read it as hello how’s it going. Normally when you read text with a screen reader and the text has line breaks you have to swipe to access the next line. And as for the question of which screen reader I tried. I tried it with both commentary screen reader and talkback. If you want to know what I mean use talkback on the Reddit website and read a nosleep post. After you do that, read the same post inside of your client. You’ll get what I mean

**Comment by QuantumBadger** [comment]
Thanks! I've added a new preference in Alpha 251, under Settings > Accessibility > Separate body text lines.

This is turned off by default, but if you check the box, each line in comments and self posts should be selectable individually.

I've included a link to Alpha 251 below, in case you want to try it out and let me know if it works!

https://redreader.org/alpha/builds/RedReader_Alpha_251.apk
